src/test/java/org/skife/jdbi/v2/sqlobject/BindInTest.java

```diff
@@ -0,0 +1,184 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.skife.jdbi.v2.sqlobject;
+
+import org.h2.jdbcx.JdbcDataSource;
+import org.junit.*;
+import org.skife.jdbi.v2.DBI;
+import org.skife.jdbi.v2.Handle;
+import org.skife.jdbi.v2.Something;
+import org.skife.jdbi.v2.sqlobject.stringtemplate.UseStringTemplate3StatementLocator;
+import org.skife.jdbi.v2.unstable.BindIn;
+
+import java.util.ArrayList;
+import java.util.Iterator;
+import java.util.List;
+import java.util.UUID;
+
+public class BindInTest
+{
+    private static Handle handle;
+
+    @BeforeClass
+    public static void init()
+    {
+        final JdbcDataSource ds = new JdbcDataSource();
```

Comment:
https://github.com/jdbi/jdbi/blob/jdbi3/core/src/test/java/org/jdbi/v3/core/H2DatabaseRule.java

used like this

https://github.com/jdbi/jdbi/blob/jdbi3/core/src/test/java/org/jdbi/v3/core/TestUuid.java#L26-L27